Sources/OktaIdx/Internal/Implementations/Version1/Responses/Responses.swift

```diff
@@ -263,4 +263,62 @@ extension IDXClient.APIVersion1 {
         let refreshToken: String?
         let idToken: String?
     }
+
+    struct Redirect {
+        enum Query {
```

Comment:
@mikenachbaur-okta What do you think about this way to manage hard-coded constans?

Reply:
Yes, I like it. It's a nice way to structure constants to make it more explicit what they're used for.